examples/test/test_resnet_ptq_cpu.py

```diff
@@ -30,6 +32,10 @@ def setup():
 @pytest.mark.parametrize("execution_order", ["pass-by-pass"])
 @pytest.mark.parametrize("system", ["local_system", "aml_system"])
 @pytest.mark.parametrize("olive_json", ["resnet_ptq_cpu.json", "resnet_ptq_cpu_aml_dataset.json"])
+@pytest.mark.skipif(
+    version.parse(OrtVersion) == version.parse("1.16.0"),
```

Comment:
maybe this issue?
https://github.com/microsoft/onnxruntime/issues/17619

Reply:
yes